src/qml/editorwidgets/ExternalResource.qml

```diff
@@ -20,7 +20,20 @@ Item {
   property ViewStatus __viewStatus
 
   //on all mimetypes image/... and on empty values it should appear as an image widget
```

Comment:
```suggestion
  //on all mimetypes "image/..." and on empty values it should appear as an image widget except when it's configured as a link
```
I think Matthias mentioned already. Just add this part, to have everything here.